src/main/java/mServer/crawler/sender/MediathekArd.java

```diff
@@ -29,15 +32,27 @@
 import mServer.crawler.CrawlerTool;
 import mServer.crawler.FilmeSuchen;
 import mServer.crawler.GetUrl;
+import mServer.crawler.sender.newsearch.Qualities;
+import mServer.tool.M3U8Utils;
+
+import org.apache.commons.lang3.StringUtils;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
-public class MediathekArd extends MediathekReader {
 
-    public final static String SENDERNAME = Const.ARD;
+public class MediathekArd extends MediathekReader {
+    private static final Logger LOG = LogManager.getLogger(MediathekArd.class);
+    private final static String SENDERNAME = Const.ARD;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Reorder the modifiers to comply with the Java Language Specification. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AModifiersOrderCheck)